QSB/Animation/AnimationType.cs

```diff
@@ -2,12 +2,12 @@
 {
 	public enum AnimationType
 	{
-		Chert,
-		Esker,
-		Feldspar,
-		Gabbro,
-		PlayerSuited,
-		PlayerUnsuited,
-		Riebeck
+		Chert = 0,
+		Esker = 1,
+		Feldspar = 2,
+		Gabbro = 3,
+		PlayerSuited = 4,
+		PlayerUnsuited = 5,
+		Riebeck = 6
```

Comment:
This is done automatically though? no need to write unneeded stuff

Reply:
it's not needed, you're right. But, maybe in the future, someone builds on top of QSB, and if you reorder enum values without explicit number values, it'll change stuff for them. Feel free to remove the numbering :)